app/models/framework/definition/ast/semantic_checker.rb

```diff
@@ -57,13 +57,14 @@ def raise_when_dependent_reference_has_mismatched_arity(field)
 
             raise Transpiler::Error,
                   "'#{field.sheet_name}' depends on #{num_dependent_fields} fields " \
-                  "#{format_string_list(field.dependent_fields)} but contains a match on " \
-                  "#{key.size} values #{format_string_list(key)}"
+                  "#{format_list(field.dependent_fields)} but contains a match on " \
+                  "#{key.size} values #{format_list(key)}"
           end
         end
 
-        def format_string_list(array)
-          '(' + array.map { |value| "'#{value}'" }.join(', ') + ')'
+        def format_list(array)
+          values = array.map { |value| value.is_a?(String) ? "'#{value}'" : value }
```

Comment:
Is this change needed because `value` may be an instance of `Any` now?

If so, since `Any` responds to `to_s` will it not be called automatically by the interpolation?

Reply:
It would be, but I wasn't sure how to get the single quotes around string values but not around the `*` from the `Any`. Is there a better way of doing this?